OT_SCHEMA_VERSION

```diff
@@ -1 +1 @@
-2.4.0
\ No newline at end of file
+022fabb
```

Comment:
This is the commit hash that includes the new field, as I don't think a new tagged version will be released in time...